documentation/staging/content/userguide/managing-domains/upgrade-domain-resource/_index.md

```diff
@@ -0,0 +1,60 @@
++++
+title = "Upgrade Domain resource"
+date = 2019-02-23T16:43:45-05:00
+weight = 2.5
+pre = "<b> </b>"
++++
+
+### Contents
+
+ - [Operator 4.0 domain resource API version change](#operator-40-domain-resource-api-version-change)
+ - [Automated upgrade of WKO 3.x domain resource](#automated-upgrade-of-weblogicoraclev8-schema-domain-resource)
+ - [Upgrade the WKOin operator version 4.0 3.x domain resource manually](#upgrade-the-weblogicoraclev8-schema-domain-resource-manually)
+
+### Operator 4.0 domain resource API version change 
+The Domain CustomResourceDefinition in Operator version 4.0 has changed significantly from previous Operator releases. For this reason, we have updated the API version of the Domain custom resource in CRD to `weblogic.oracle/v9` from `weblogic.oracle/v8`. We continue to support the Domains with API version `weblogic.oracle/v8` and provide full backward compatibility. If you want to use new fields introduced in the latest `weblogic.oracle/v9` schema, then you will need to update the API version in your Domain resource YAML. 
+
+### Automated upgrade of `weblogic.oracle/v8` schema domain resource
+The 4.0 Operator provides a seamless upgrade of the Domain resources with `weblogic.oracle/v8` version of the schema. When you create a Domain using a domain resource YAML with `weblogic.oracle/v8` schema in a namespace managed by the WKO 4.0, the [WebLogic Domain resource conversion webhook]({{< relref "/userguide/managing-operators/conversion-webhook.md" >}}) performs an automated upgrade of the domain resource to the `weblogic.oracle/v9` schema. The conversion webhook runtime converts the `weblogic.oracle/v8` configuration to the equivalent configuration in WKO 4.0. Similarly, when [upgrading the Operator version]({{< relref "/userguide/managing-operators/installation#upgrade-the-operator" >}}), Domains resources with `weblogic.oracle/v8` schema are seamlessly upgraded.
```

Comment:
IMO, this is a generic section for automated Domain upgrade and not specific to auxiliary images. I have added a note in the auxiliary image "automated upgrade" section to mention the conversion to low-level Kubernetes schema. Also, I also included the note about the low-level schema conversion in the manual upgrade section below (in this document) since the auxiliary image manual upgrade section provides a link to it.

Reply:
ok, I have added the note.